gradle/validation/precommit.gradle

```diff
@@ -26,7 +26,6 @@ configure(rootProject) {
     dependsOn ":verifyLocks"
     dependsOn ":versionsPropsAreSorted"
     dependsOn ":checkWorkingCopyClean"
-    dependsOn ":validateSourcePatterns"
```

Comment:
I think we can leave the dependency on the root task as it depends on other source pattern validations. Or remove the snippet that connects them later.

Reply:
I wanted to make it consistent with the others. Now all are handled the same.

Reply:
Removing the connection in subprojects is not good, as now I can run `gradlew check` in a subproject and it does everything right.

Reply:
Fair enough. But then the top-level one doesn't need to depend on children?

Reply:
Ah, ok. See it now.